rust/src/lib.rs

```diff
@@ -3673,7 +3681,7 @@ pub unsafe extern "C" fn zcashlc_tor_lwd_conn_check_single_use_taddr(
             if !found {
                 let blocks_since_exposure = match meta.exposure() {
                     Exposure::Exposed { at_height, .. } => {
-                        f64::from(std::cmp::max(cur_height - at_height, 1))
+                        f64::from(std::cmp::max(u32::from(cur_height - at_height), 1))
```

Comment:
Undo this revert of my clippy lint fix from https://github.com/Electric-Coin-Company/zcash-light-client-ffi/pull/256/commits/fee34a1e12ed78f1dc869c3b4b7fbfaff184ee24 (`BlockHeight - BlockHeight` produces `u32` already).

Reply:
Huh, weird, not sure that got in.